stories/components/OrgAvatar.tsx

```diff
@@ -0,0 +1,88 @@
+import { FC } from "react"
+import styled from "styled-components"
+
+const ImageContainer = styled.div`
+  width: 87px;
+  height: 87px;
+  border: 2.5px solid var(--bs-green); //dependant on the stance
```

Comment:
nit: switch this to `border-width: 2.5px` and omit the comment, to separate the static css from the stuff configured in the component

Reply:
``` border: 2.5px solid ```
I switched this to above because we still needed the border type